src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_ThrowStatement.cs

```diff
@@ -12,7 +14,7 @@ internal sealed partial class LocalRewriter
     {
         public override BoundNode VisitThrowStatement(BoundThrowStatement node)
         {
-            var result = (BoundStatement)base.VisitThrowStatement(node);
+            var result = (BoundStatement)base.VisitThrowStatement(node)!;
```

Comment:
Assert?

Reply:
Base visitors never return null.  Repeatedly asserting that isn't worth it.

---
In reply to: [390545327](https://github.com/dotnet/roslyn/pull/42068#discussion_r390545327) [](ancestors = 390545327)